x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -10,11 +10,15 @@ import { docLinks } from '../shared/doc_links';
 export const DOCS_PREFIX = docLinks.appSearchBase;
 
 export const ROOT_PATH = '/';
+export const NOT_FOUND_PATH = '/404';
 export const SETUP_GUIDE_PATH = '/setup_guide';
 export const LIBRARY_PATH = '/library';
 export const SETTINGS_PATH = '/settings/account';
 export const CREDENTIALS_PATH = '/credentials';
-export const ROLE_MAPPINGS_PATH = '#/role-mappings'; // This page seems to 404 if the # isn't included
+
+export const ROLE_MAPPINGS_PATH = '/role-mappings';
+export const ROLE_MAPPING_PATH = `${ROLE_MAPPINGS_PATH}/:roleId/edit`;
```

Comment:
[Not a blocker, just throwing this out there] Do we like the extra `/edit` at the end of this route? I'm wondering if we should consider just simplifying to `/role_mappings/{some-id}` - no worries if we don't like that though

Reply:
I was thinking of doing this and you pushed me over the edge. Thanks! Just an FYI that there are other places in this file that use both `/edit` and hyphens in the URLs.

Reply:
The hyphens should be URLs pointing back to ent-search right now (those are an unfortunate leftover from back when I thought the Kibana plugin and the standalone UI were supposed to live side-by-side). Those _should_ be gone by the time we finish the migration 🤞 

I'll make a note go back in and do a pass for `/edit` URLs though! I already sneakily removed mine from my Curation route :trollface: